weaviate/values.yaml

```diff
@@ -561,6 +561,39 @@ modules:
         # enable if running with CUDA support
         # nvidia.com/gpu: 1
 
+  # The sum-transformers module makes result texts summarizations
+  sum-transformers:
+    enabled: false
+    tag: facebook-bart-large-cnn-1.0.0
```

Comment:
What I think is missing from all the `modules` is the `inferenceUrl`. Maybe it would be a good idea to add it as a comment.

```yaml
    enable: false
    # You can set directly an inference URL of this module without deploying it with this release.
    # You can do so by setting a value for the `inferenceUrl` here AND by setting the `enable` to `false`
    inferenceUrl: {}
```

Reply:
Commented works too, but I think we need it for ALL modules.